### PR TITLE
feat(swc-angular): ✨ upgrade @swc/core to 1.6.13

### LINKED ISSUE #557: Add support for @swc/core latest packages
Currently I got error:

```bash
@swc/core version 1.7.21 is incompatible with @jscutlery/swc-angular.
    Please use @swc/core version 1.4.x or 1.5.x.
    > npm add -D @swc/core@~1.5.0
```

would be good if it will works with latest versions of @swc/core

Also I'm debugging your package, it's working with no problem with @swc/core 1.6.x but broken in 1.7.x

### PR DESCRIPTION
**Pull request by pumano**
closes #557

### CODE REVIEW

File: packages/swc-angular/src/index.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ function assertCompatibleSwcCoreVersion(version: string) {
     }>(packageJsonPath).version;
   }
 
-  if (!(version.startsWith('1.4.') || version.startsWith('1.5.'))) {
+  if (!(version.startsWith('1.4.') || version.startsWith('1.5.') || version.startsWith('1.6.'))) {
     console.error(`
     @swc/core version ${version} is incompatible with @jscutlery/swc-angular.
     Please use @swc/core version 1.4.x or 1.5.x.
```
Comment (edbzn) [inline packages/swc-angular/src/index.ts:94]: We should update the log as well: 
```suggestion
    Please use @swc/core version 1.4.x or 1.5.x. or 1.6.x.
```

Comment (pumano) [reply packages/swc-angular/src/index.ts:94]: Ok I will apply suggestion soon.

Comment (edbzn) [reply packages/swc-angular/src/index.ts]: I just did it I was on your branch, thanks for the work @pumano!

**Comment by pumano** [comment]
@edbzn looks like that component testa broken somehow

**Comment by edbzn** [comment]
Yeah there's false positive with visual testing, I'm on it.

**Pull request merged**